README.md

```diff
@@ -504,4 +504,200 @@ Current the following 5 log levels are supported:
+Sample response:
+
+```python
+{
+ 'records': [
+     {
+         'fields': {
+             'card_number': '4111111111111111',
+             'cvv': '127',
+             'expiry_date': '11/35',
+             'fullname': 'monica',
+             'skyflow_id': 'f8d8a622-b557-4c6b-a12c-c5ebe0b0bfd9'
+         },
+         'table': 'cards'
+     },
+     {
+         'fields': {
+             'card_number': '123456',
+             'cvv': '317',
+             'expiry_date': '10/23',
+             'fullname': 'sam',
+             'skyflow_id': 'da26de53-95d5-4bdb-99db-8d8c66a35ff9'
+         },
+         'table': 'cards'
+     }
+ ],
+ 'errors': [
+     {
+         'error': {
+             'code': '404',
+             'description': 'No Records Found'
+         },
+         'skyflow_ids': ['invalid skyflow id']
+     }
+ ]
+}
+```
+
+### Update
+
+To update data in your vault, use the `update(records: dict, options: UpdateOptions)` method. The `records` parameter takes a Dictionary that contains records to fetch. If `UpdateTokens` is `True`, Skyflow returns tokens for the record you just updated. If `, ids if `UpdateOptions` is `False`, Skyflow returns IDs for the record you updated.
+
+```python
+# Optional, indicates whether to return all fields for updated data. Defaults to 'true'.
+options: UpdateOptions
+```
+
+```python
+{
+       'records': [
+           {
+               'id': str, # Skyflow ID of the record to be updated.
+               'table': str, # Name of table holding the skyflowID.
+               'fields': {
+                  str: str # Name of the column and value to update.
+               }
+           }
+       ]
+}
+```
+Sample usage
+
+The following snippet shows how to use the `update()` method. For details, see [update_sample.py](https://github.com/skyflowapi/skyflow-python/blob/main/samples/update_sample.py),
+
+```python
+records = {
+       'records': [
+           {
+               'id': '56513264-fc45-41fa-9cb0-d1ad3602bc49',
+               'table': 'cards',
+               'fields': {
+                   'card_number': '45678910234'
+               }
+           }
+       ]
+   }
+try:
+   client.update(records, UpdateOptions(True))
+except SkyflowError as e:
+   if e.data:
+       print(e.data)
+   else:
+       print(e)
+```
+
+Sample response
+
+`UpdateOptions` set to `True`
+
+```python
+{
+ 'records':[
+   {
+     'id':'56513264-fc45-41fa-9cb0-d1ad3602bc49',
+     'fields':{
+       'card_number':'0051-6502-5704-9879'
+     }
+   }
+ ],
+ 'errors':[]
+}
+```
+
+`UpdateOptions` set to `False`
+
+```python
+{
+ 'records':[
+   {
+     'id':'56513264-fc45-41fa-9cb0-d1ad3602bc49'
+   }
+ ],
+ 'errors':[]
+}
+```
+
+Sample Error
+
+```python
+{
+ 'records':[
+   {
+     'id':'56513264-fc45-41fa-9cb0-d1ad3602bc49'
+   }
+ ],
+ 'errors':[
+   {
+     'error':{
+       'code':404,
+       'description':'Token for skyflowID  doesn"t exist in vault - Request ID: a8def196-9569-9cb7-9974-f899f9e4bd0a'
+     }
+   }
+ ]
+}
+```
```

Comment:
This content should replace the Get by ID section and have equivalent heading levels.

Reply:
made changes